main.py

```diff
@@ -44,6 +49,34 @@ def tba_webhook():
     return msg_type
 
 
+@app.route('/predictions')
+def give_predictions():
+    return str(trueskill_predictions)
```

Comment:
Look, I don't understand why this is here, or why this needs to be here, or why you feel the need to expose our Slack data to those who don't have access to our Slack.